cumulusci/tasks/release_notes/generator.py

```diff
@@ -136,6 +146,21 @@ def _get_release(self):
                 "Release not found for tag: {}".format(self.current_tag)
             )
 
+    def _render_empty_pr_section(self):
+        section_lines = []
+        if self.empty_pull_requests:
+            section_lines.append("\n# Pull requests with no release notes")
+            for content in self.empty_pull_requests:
+                section_lines.append(
+                    "\n* {}".format(self._mark_down_link_to_pr(content))
+                )
+
+        return section_lines
+
+    def _mark_down_link_to_pr(self, pull_request):
+        if pull_request:
```

Comment:
When would this line be false?

Reply:
I don't think it ever would be. Removing :)